subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -191,6 +191,8 @@ dependencies {
         api(libs.jtar)                  { version { strictly("2.3") }}
         api(libs.kotlinCoroutines)      { version { strictly("1.5.2") }}
         api(libs.kotlinCoroutinesDebug) { version { strictly("1.5.2") }}
+        api(libs.kotlinxSerializationCore)   { version { strictly("1.6.3") }}
```

Comment:
⚠️  We have updated the dependency version here. I am not sure if this was intended.
These dependencies are also unrelated to any of the JSON writer logic we are refactoring, right?

Reply:
That is true, I ended up changing it, because I thought of using it for the `JsonWriter`.